llvm/lib/Target/AMDGPU/BUFInstructions.td

```diff
@@ -1699,7 +1709,7 @@ multiclass SIBufferAtomicPat_Common<string OpPrefix, ValueType vt, string Inst,
 
 multiclass SIBufferAtomicPat<string OpPrefix, ValueType vt, string Inst,
                              list<string> RtnModes = ["ret", "noret"]> {
-  let SubtargetPredicate = HasUnrestrictedSOffset in {
+  let OtherPredicates = [HasUnrestrictedSOffset] in {
```

Comment:
A side question, what is the difference between `OtherPredicates` and `SubtargetPredicate`? It looks like you swapped a couple of them here.

Reply:
SubtargetPredicate is just one thing, and OtherPredicates is a list? The usage pattern consistency has been declining for a while